feature-libs/product-configurator/rulebased/components/show-more/configurator-show-more.component.ts

```diff
@@ -37,9 +40,13 @@ export class ConfiguratorShowMoreComponent implements AfterViewInit {
     this.showHiddenText$.next(!this.showHiddenText$.value);
 
     this.showHiddenText$.value
-      ? (this.textToShow = this.text)
-      : (this.textToShow = this.text.substring(0, this.textSize));
+      ? (this.textToShow = this.textNormalized)
+      : (this.textToShow = this.textNormalized.substring(0, this.textSize));
 
     this.cdRef.detectChanges();
   }
+
+  protected normalize(text: string = ''): string {
```

Comment:
check if we already have such helper function somewhere. looks rather generic and might be reused

Reply:
we have similar function in one core component, but it is protected and exported